activerecord/lib/active_record/validations.rb

```diff
@@ -82,6 +82,21 @@ def perform_validations(options = {}) # :nodoc:
       options[:validate] == false || valid?(options[:context])
     end
   end
+
+  module Persistence
```

Comment:
Is there a way to maintain this functionality without changing `Persistence` methods in `Validations`? 😬